templates/_macros.html

```diff
@@ -0,0 +1,108 @@
+{#-#################################################################
+<File Name>
+  _macros.html
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  May 05, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Generic template macros used in other templates,
+  see http://jinja.pocoo.org/docs/2.9/templates/#macros for more information
+  on how to use macros.
+
+  In some cases macros are passed to to macros using the special caller
+  variable, see http://jinja.pocoo.org/docs/2.9/templates/#call
+
+<Overview of macros>
+  option_grid()
+    Used to display software supply chain tools (name and logo) in a grid.
+    If an option (tool) is selected (clicked) a form group is expanded below,
+    usually showing the common command for that tool and additional information
+    which can be modified by the user. Used in versioning, building, quality
+    and packaging templates.
+
+  form_container()
+    Used to list all the form groups that get actually posted to the client.
+    Form groups can be copied from the option grid and individually added,
+    removed and re-ordered. Used in versioning, building, quality and
+    packaging templates.
+
+  comment_form()
+    Displays a textarea form field used for user comments. Used in
+    form_container macro and all templates that require user interaction.
+
+  progress_bar()
+    Used on all pages to give user an idea of how much work is left to do.
+
+#################################################################-#}
+
+{% macro option_grid(options) %}
+<div class="opt-row row no-gutters">
+  <div class="opt-cell col opt-cell-top">
+    <div class="opt-content">
+      <div class="opt-text text-center">Custom Command</div>
```

Comment:
The `Custom Command` field is listed before the other popular version control systems.  Which option is the user likely to select?  If it's not `Custom Command`, then it should be listed after the other popular options.

Before clicking any of the buttons, I thought the user was expected to enter the custom command and then select an icon.

What if instead of a long button for the custom command (which looks strange to me and is different in style from the other buttons) you provide an "other" button (of the same size as the others)?

Reply:
The idea was to make the "custom command" button easily recognizable among all the other buttons. But anther user has suggested something very similar, so I will probably change this. :)

Reply:
Will be fixed with https://github.com/in-toto/layout-web-tool/issues/25#issuecomment-329879234